packages/tasks/src/tasks/image-to-image/inference.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+export type ImageToImageOutput = unknown[];
+
+/**
+ * Inputs for Image To Image inference
+ */
+export interface ImageToImageInput {
+	/**
+	 * One or more images to generate images from
+	 */
+	inputs: unknown;
```

Comment:
just as audio, my understanding is that the image is passed through data rather than params